toolset/benchmark/utils.py

```diff
@@ -138,3 +139,36 @@ def header(message, top='-', bottom='-'):
       else:
         result += "\n%s" % bottomheader
     return result + '\n'
+
+def check_services(services):
+
+  def check_service(address, port):
+    try:
+      s = socket.socket()
+      s.settimeout(5)
```

Comment:
This probably needs to be at least 15 to accommodate databases like mongodb that are wickedly slow on the restart